packages/polaris-viz-core/src/components/SparkBarSeries/SparkBarSeries.tsx

```diff
@@ -133,7 +131,7 @@ export function SparkBarSeries({
           strokeLinecap="round"
           opacity="0.9"
           strokeDashoffset={strokeDashoffset}
-          strokeDasharray={strokeDasharray}
+          strokeDasharray="0.1 4"
```

Comment:
I'm not sure if this is the best option to hardcode this value but we need to match the same value as in the `LineChart` and the `SparkLineChart`

<img width="350" alt="Screenshot 2022-11-23 at 11 07 13 AM" src="https://user-images.githubusercontent.com/64446645/203628062-d7f774ed-5e1e-42b5-817d-04175477632c.png">

Reply:
Can you make this constant we can share between `LineChart` and `SparkLineChart`?